src/@commonsku/styles/Sizer.tsx

```diff
@@ -32,8 +32,7 @@ export type SizerTypes = {
 };
 
 export const SizerCss = css<SizerTypes>`
-    padding: ${(props) => props.padded ? '0.5rem' : 'initial'};
-    margin-left: ${(props) => (props.offset || 0)/12 * 100}%;
+    ${(props) => props.offset && `margin-left: ${(props.offset || 0)/12 * 100}%;`}
```

Comment:
if we already have "props.offset &&", why we still need "props.offset ||" ?

Reply:
Yes, I've corrected it right now